packit/base_git.py

```diff
@@ -225,7 +225,13 @@ def with_action(self, action: ActionName) -> bool:
         if action in self.package_config.actions:
             command = self.package_config.actions[action]
             logger.info(f"Using user-defined script for {action}: {command}")
-            utils.run_command(cmd=command, cwd=self.local_project.working_dir)
+            if isinstance(command, str):
+                commands = [command]
+            else:
+                commands = command
+            logger.debug(commands)
```

Comment:
We already log it a few lines above, so this doesn't add any useful info.
I'd rather move the
```python
logger.info(f"Using user-defined script for {action}: {commands}")
```
here, instead of this `logger.debug(commands)`

Reply:
please address also this one